document/src/main/java/com/ritense/document/web/rest/impl/JsonSchemaDocumentResource.java

```diff
@@ -127,9 +128,20 @@ public ResponseEntity<Void> assignHandlerToDocument(
         }
     }
 
+    @Override
+    @PostMapping("/v1/document/assign")
+    public ResponseEntity<Void> assignHandlerToDocuments(@RequestBody AssignToDocumentsRequest request) {
+        try {
+            documentService.assignUserToDocuments(request.getDocumentIds(), request.getAssigneeId());
+            return ResponseEntity.ok().build();
+        } catch (Exception e) {
```

Comment:
This is a strange one. Did you mean to do this for any particular exception, or just in general?

Reply:
Fixed